content/Rustup-1.28.2.md

```diff
@@ -0,0 +1,85 @@
++++
+path = "2025/04/25/Rustup-1.28.2"
+title = "Announcing rustup 1.28.2"
+authors = ["The Rustup Team"]
+aliases = ["2025/04/25/Rustup-1.28.2.html"]
++++
+
+The rustup team is happy to announce the release of rustup version 1.28.2.
+[Rustup][install] is the recommended tool to install [Rust][rust], a programming language that is
+empowering everyone to build reliable and efficient software.
+
+## What's new in rustup 1.28.2
+
+The headlines of this release are:
+
+- The cURL download backend is now officially deprecated and a warning will start to show up
+  when it is used. [pr#4277]
+
+  - While rustup predates reqwest and rustls, the rustup team has long wanted to standardize on
+    an HTTP + TLS stack in Rust, which should increase security, potentially improve performance, and
```

Comment:
Is the actual stack now all in Rust? I see `native-tls` in the Cargo.lock file, which presumably is non-Rust code.

Reply:
It is not -- reqwest + rustls is now the default but in addition to the cURL fallback there is still a native-tls fallback option.

Reply:
@shepmaster Thanks! I think I can change the ending to "... maintaining the reqwest-based stack". A Rust-based stack is our long-term goal but probably it's too early to say that right now.

Reply:
> there is still a native-tls fallback

Is that intended to be deprecated eventually as well? If not, then it seems like the overall goal isn't "there is only a pure-Rust stack". Since the benefit of "a pure-Rust stack by default" has already occurred, it seems that this change is purely about reducing maintenance burden (still a good thing!).

Reply:
@shepmaster IIRC the deprecation of `native-tls` is indeed on our current project roadmap.

Reply:
Depending on how far out you think that may be, it could be worth a tiny mention here and now. That way, people don't jump from cURL to native-tls only to need to switch again, at least without trying the intended stack first (and filing bugs if needed 🤞)

Reply:
Fair point; https://github.com/rust-lang/rustup/pull/4317.

Reply:
@djc I have mirrored your change in this PR; I'll soon do the same in https://github.com/rust-lang/rustup/pull/4313.

Reply:
@Mark-Simulacrum Good point! I've adjusted the phrasing.